wfl/fit/ace.py

```diff
@@ -118,9 +189,8 @@ def _check_output_files(file_base, formats):
 
         xpr = ExPyRe(name=remote_info.job_name, pre_run_commands=remote_info.pre_cmds, post_run_commands=remote_info.post_cmds,
                       env_vars=remote_info.env_vars, input_files=input_files, output_files=output_files, function=fit,
-                      kwargs= {'fitting_configs': fitting_configs, 'ACE_name' : ACE_name, 'params': params,
-                               'ref_property_prefix': ref_property_prefix,
-                               'run_dir': str(run_dir), 'formats': formats, 'ace_fit_exec': ace_fit_exec,
+                      kwargs= {'fitting_configs': fitting_configs, 'ace_fit_params': ace_fit_params,
+                               'run_dir': str(run_dir), 'ace_fit_exec': ace_fit_exec,
```

Comment:
What happened to `REF_property_prefix`?

Reply:
That is managed in `prepare_params()`. `fit()` doesn't modify the params beyond adding `.xyz` to fit to and optionally a `run_dir`.

Reply:
My first instinct is to say that it belongs in the `fit()` function, but I'll take a look at the revised version and think about this.

Reply:
OK, I understand what it's doing.  I'm definitely not sold on the fact that now you need to call 3 separate routines to do a fit.  At least I'd like a function that calls all 3 (prep params, prep configs, fit). I'd actually argue that this function should be called `fit`, like the GAP fitting codes, and this one that expects already prepared params and configs should be called something more specific.

Reply:
okay, I thought there should be a function that doesn't modify the parameters beyond what's necessary. Renamed it to   `run_ace_fit`, following the GAP example. `fit()` in the next commit should behave equivalently to `fit()` currently in `main`.

Reply:
I'm happy with that.